### ISSUE TITLE
Authenticated calls fail.

### ISSUE THREAD

**Issue by Davincij15** [LanderV/CryptsyAPICS#3]
I keep getting the following error....

sResult = "{\"success\":\"0\",\"error\":\"Request Method Must Be Post\"}"

I have tried so many example codes and asked contacted support with no luck.   If you can get this to work I will flip you a bitcoin.  I have spent a week trying to find code that works with no luck not even python code works.

Cheers

**Comment by LanderV** [comment]
Hi, 

Cryptsy changed their API a few times so this code is probably a bit outdated. I can help you getting it working again. Please contact me at contact AT lavasolutions .eu 

Regards, 
Lander 

----- Original Message -----

**Comment by Davincij15** [comment]
Figured it out! Using Fiddler man in the middle proxy server.
1. When a C# developer uses "https://cryptsy.com"  it gets redirected to www.cryptsy.com however C# does not send the "POST" it sends a "GET" even though my original call to  "https://cryptsy.com" was a post.  But this is why I kept getting  **{"success":"0","error":"Request Method Must Be Post"}**
2. Calling directly www.cryptsy.com or api.cryptsy.com gives an error 417 using C# because .NET send a "Expect: 100-Continue" in the HTTP header data thus some servers choke on this so you must add this into your code...

`System.Net.ServicePointManager.Expect100Continue = false;`

So that's why I could not get it to work but it's working fine now.

Cheers

Davinci
